README.md

```diff
@@ -123,14 +123,14 @@ What Woketo does _not_ do?
 
 Currently there is no support of the following:
 
-- WebSocket extensions, currently not supported but will probably be in the future
-- [WAMP](http://wamp-proto.org/) implementation will probably never be done by Woketo itself as it's a layer on top of
+- WebSocket extensions, currently not supported but probably will be in the future
+- [WAMP](http://wamp-proto.org/) implementation might never be done by Woketo itself provided it's a layer on top of
   WebSockets. This includes JSON-RPC and other layers up to WebSockets.
-- Woketo have no HTTP server layer and is not compatible with [PHP PM](https://github.com/php-pm/php-pm). This is planned for next releases (probably the v3).
+- Woketo does not have an HTTP server layer and is not compatible with [PHP PM](https://github.com/php-pm/php-pm). This is planned in the future. (probably the v3).
```

Comment:
This may be change because the compatibility with php-pm is not gonna happen finaly. But http layer will probably come some day.